spec/i18n_spec.rb

```diff
@@ -20,4 +20,34 @@
       "#{unused_keys.leaves.count} unused i18n keys, run `i18n-tasks unused' to show them"
     )
   end
+
+  root_dir = File.expand_path(File.join(File.dirname(__FILE__), '../'))
+
+  Dir[File.join(root_dir, '/config/locales/**/*.yml')].each do |full_path|
+    i18n_file = full_path.sub("#{root_dir}/", '')
+
+    describe i18n_file do
+      it 'has only lower_snake_case keys' do
+        keys = hash_keys(YAML.load_file(full_path))
+
+        bad_keys = keys.reject { |key| key =~ /^[a-z0-9_.]+$/ }
```

Comment:
I'm OK not worrying about those cases?

The things that caused us problems in this most recent round were spaces, quotes and dashes in keys, so I just wanted to handle those. A few extra underscores seems ok for now to me.